libnopegl/src/hud.c

```diff
@@ -1390,6 +1390,9 @@ void ngli_hud_draw(struct hud *s)
         ctx->render_pass_started = 1;
     }
 
+    ngli_gpu_ctx_set_viewport(gpu_ctx, &ctx->viewport);
+    ngli_gpu_ctx_set_scissor(gpu_ctx, &ctx->scissor);
```

Comment:
This should be squashed in another commit (the hash doesn't exist)

Reply:
Done